frameworks/elastic/tests/test_shakedown.py

```diff
@@ -151,7 +150,28 @@ def test_master_node_replace():
     # Ideally, the pod will get placed on a different agent. This test will verify that the remaining two masters
     # find the replaced master at its new IP address. This requires a reasonably low TTL for Java DNS lookups.
     cmd.run_cli('elastic --name={} pod replace master-0'.format(FOLDERED_SERVICE_NAME))
-    # setup_function will verify that the cluster becomes healthy again.
+    # pre_test_setup will verify that the cluster becomes healthy again.
```

Comment:
The current logs do connect the `pre_test_setup` with the appropriate failing test: ```
[08:57:21]F:		 [frameworks.elastic.tests.test_shakedown.test_bump_node_counts] test setup failed
[08:57:21]F:		 [frameworks.elastic.tests.test_shakedown.test_bump_node_counts] @pytest.fixture(autouse=True)
    def pre_test_setup():
>       sdk_tasks.check_running(FOLDERED_SERVICE_NAME, config.DEFAULT_TASK_COUNT)
```